includes/transformer/class-base.php

```diff
@@ -84,11 +96,12 @@ public function to_activity( $type ) {
 
 		$activity = new Activity();
 		$activity->set_type( $type );
-		$activity->set_object( $object );
```

Comment:
this should be as is, because it pre-fills the Activity with data and it will no longer be done with your change.